### ISSUE TITLE
Custom CSS loaded but not rendered

### ISSUE THREAD

**Issue by mrclschstr** [mrpbennett/catppucin-homer#6]
At the moment I can't get the theme to work and I don't really know what I'm doing wrong. I have installed the theme according to the instructions and I can also see in the Firefox Developer Tools that the CSS is loaded correctly. However, nothing changes in the design of the page. See also: https://github.com/bastienwirtz/homer/issues/565#issuecomment-2265217952

I would be very happy about any support.

**Addition:** The problems only occur with the `latest` tag of homer. If I use the tag `v24.05.1`, everything works as expected! Could this be related to the revised theming system? See https://github.com/bastienwirtz/homer/commit/359f1ffb53cdb3476acbfcb8c1af70dff2eaf403

**Comment by mrpbennett** [comment]
I would very much say this is related to the new theming system. I will take a look and adjust when I have some time.

**Comment by Noste92** [comment]
Hi, they changed the css classes names of the dark and light theme I opened a PR to fix it.

**Comment by mrpbennett** [comment]
amazing thanks...merged your PR.

**Comment by l0f3n** [comment]
I cannot get this to work. The CSS is loading but nothing on the page is changing. Changing theme does nothing.

**Comment by mrpbennett** [comment]
@l0f3n I have just tested this and added the style sheet from the repo inside the `assets/themes` directory then added the following to the `config.yaml`

```
stylesheet:
  - "assets/themes/catppuccin-macchiato.css"
```

**Comment by l0f3n** [comment]
Thanks for the reply. I will provide more info. I am using this docker image: https://hub.docker.com/r/b4bz/homer 

`config.yml`:
```yml
stylesheet:
  - "assets/themes/catppuccin-mocha.css"
```

And in the browser I see, with the expected content in the response.
![image](https://github.com/user-attachments/assets/69092811-2cda-4d88-a789-125c43a013d9)

**Comment by mrpbennett** [comment]
Yup this is the case, I have tested it and the theme changed. I have just added a docker compose to the repo for testing

**Comment by l0f3n** [comment]
Yeah, sorry. Its working. I wanted to get it to look your screenshots, but my corners are a lot more rounded. I see now that this is a just a color-scheme and does not change other styling. What theme are you using?

**Comment by l0f3n** [comment]
But now one of my fa icon `fa-film` turns the wrong color in darkmode and becomes invisible. I can see it in light mode.

The `--highlight-primary` variable seems to become transparent when dark mode is turned on, that is why it disappears, but it should be teal. I dont know if it affects something else.

**Comment by l0f3n** [comment]
@mrpbennett Sorry for the spam. No issues relating this, I was just being dumb. You can close this. But I found another problem and created a new issue #9
